tasks/main.yml

```diff
@@ -125,13 +125,34 @@
   when:
     - deploydrupal_site_install
 
+# Config Value Check - This is primarily to catch configuration value updates
+# done via update hooks from Dependabot PR's. This is to replace the dev process
+# that would be done in a local environment.
```

Comment:
TIL grep exits with a non-zero value if it can’t find what you tell it to look for. Makes perfect sense, just never really thought about it.

Can we add a comment just above this that explains this? It took me a moment —and a quick Google search— to realize how this check would fail a build. That non-zero exit is the key!

Reply:
Ya I learned that during this work too.

Good call on documenting that. It's a bit too clever without some documentation.

Reply:
Documented. be74c49493c41dfe95ae4e2a3aa24a96f759e84e